src/current/cockroachcloud/authorization.md

```diff
@@ -95,9 +104,9 @@ This role can be considered a more restricted alternative to [Cluster Administra
 - Manage SQL users from the cloud console.
 - Create or delete a cluster.
 
-### Cluster Administrator
+This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited on the folder's clusters, descendent folders, and their descendants.
```

Comment:
```suggestion
This role can be granted at the scope of the organization, on an individual cluster, or on a folder. If granted on a folder, it is inherited by the folder's clusters, descendent folders, and their descendants.
```

Reply:
I think I want to leave it as "by" because a role is something that an identity has in a context. The folder is the context.